common/changes/@ducky/plumage-react/feat-status_2022-08-16-13-49.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@ducky/plumage-react",
+      "comment": "",
+      "type": "none"
```

Comment:
You should register a change for the `plumage-react` package because you do also want to release this component for use in react.